src/python/pants/rules/core/test_cloc.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from typing import Sequence
+
+from pants.backend.jvm.targets.java_library import JavaLibrary
+from pants.build_graph.build_file_aliases import BuildFileAliases
+from pants.rules.core import cloc
+from pants_test.console_rule_test_base import ConsoleRuleTestBase
+
+
+class ClocTest(ConsoleRuleTestBase):
+  goal_cls = cloc.CountLinesOfCode
+
+  @classmethod
+  def rules(cls):
+    return super().rules() + cloc.rules()
+
+  @classmethod
+  def alias_groups(cls):
+    return BuildFileAliases(targets={ 'java_library': JavaLibrary })
```

Comment:
Kill space padding in the dict literal; ie: `{ 'java_library': JavaLibrary }` -> `{'java_library': JavaLibrary}`.